GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -191,6 +192,17 @@ bool ExporterHelper::ExportProjectForPixiPreview(
     previousTime = LogTimeSpent("Events code export", previousTime);
   }
 
+  auto projectUsedResources =
+      gd::UsedResourcesFinder::FindProjectUsedResources(exportedProject);
+  std::vector<std::set<gd::String>> layersUsedResources;
```

Comment:
```suggestion
  std::vector<std::set<gd::String>> layoutsUsedResources;
```

or scenesUsedResources